PLIER_pathways_analysis.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "PLIER pathways analysis"
+output: html_notebook
+---
+
+```{r}
+meaningful_difference <- 0.2
+```
+
+### What additional pathways are identified by PLIER after doubling the sample size?
+
+Here we look for cancer pathways that are found more frequently in the data with the full sample size than in data with half the sample size.
+We ran PLIER 10 times using each combination of data normalization method and sample size (half or full).
+Out of those 10 runs, we found the proportion of times each pathway was significantly associated with a latent variable.
+We set an arbitrary "meaningful difference" threshold of `r meaningful_difference` to detect when that proportion was meaningfully greater using full sample size compared to half sample size and require array and RNA-seq data to both to satisfy the condition.
+We also arbitrarily limit results to those pathways found in over half of runs using the full size data.
```

Comment:
Say why. It seems reasonable. I expect it's because we want to focus on pathways that can be reliably detected (i.e., are "stable") in the larger sample size data.

Reply:
Added reliability and stability language